index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require 'src/db.php';
+
+require 'src/functions.php';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en" lang="vi">
+<head>
+	<title>Bookshelf</title>
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="https://kit.fontawesome.com/5139af5f3f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div tabindex='0'><h1>a bookshelf</h1></div>
+    <?php echo displayBooks($books);
+    ?>
+    <p></p>
```

Comment:
what does this p tag do?

Reply:
it's just to add an extra line below the displayed stuff, so the displayed stuff isn't touching the end of the page... I've changed it to: 

<br id="extra line to prevent display touching end of page"/>

not sure if that is ok/the best way